pkg/ui/workspaces/cluster-ui/src/components/tooltipMessages/tableMetadataLastUpdatedTooltip.tsx

```diff
@@ -49,12 +40,35 @@ const formatErrorMessage = (
   );
 };
 
+type Props = {
+  timestamp?: moment.Moment | null;
+  children: (
+    formattedRelativeTime: React.ReactNode,
+    icon?: JSX.Element,
+  ) => React.ReactNode;
+  errorMessage?: string;
+  loading?: boolean;
+};
+
 export const TableMetadataLastUpdatedTooltip = ({
   timestamp,
   errorMessage,
   children,
+  loading,
 }: Props) => {
-  const durationText = timestamp?.fromNow() ?? "Never";
+  const duration = (
+    <span>
```

Comment:
why is this wrapper necessary?

Reply:
It's to wrap the skeleton component when it's in loading state so that it doesn't go onto a new line.